react/features/overlay/components/AbstractPageReloadOverlay.js

```diff
@@ -0,0 +1,187 @@
+import React, { Component } from 'react';
+
+import { randomInt } from '../../base/util';
+
+import { reconnectNow } from '../functions';
+import ReloadButton from './ReloadButton';
+
+declare var AJS: Object;
+declare var APP: Object;
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
```

Comment:
OK. I see. 

We can have a discussion about it because I'm not aware of rule that `require` is forbidden and in this case the line looks so much prettier :) If it turns out that most of the people want import for this one than we can change them all trough another PR. WDYT?